pom.xml

```diff
@@ -33,7 +33,7 @@
                   serialization changes.
                   https://github.com/aws/aws-sdk-java-v2/issues/3127
                  -->
-                <version>2.17.81</version>
+                <version>2.17.83</version>
```

Comment:
There's a warning comment here about upgrading https://github.com/aws/aws-sdk-java-v2/issues/3127 .  Is this going to be an issue?

Reply:
yep, according to [this pr](https://github.com/aws-greengrass/aws-greengrass-nucleus/commit/d79f06959d4aef5c1ac06e19b43f6698ef643a05) awssdk version was reverted from 156 to 81 cuz the higher version was causing all our plugins to break. 83 is the first higher stable version which supports the newer ggv2-data artifact so an awssdk bump is necessary.
im still checking if this causes any of the plugins to fail through uats
however, i raised a pr in order to check if the change passes through github ci cuz i found our integ tests to be quite flaky (failed intermittently)

Reply:
If we're going to bump it, let's go all the way. Use the latest available version.

Reply:
updated sdk with 2.17.295